docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md

```diff
@@ -545,6 +545,12 @@ Create the hook execution file for Kata:
 /usr/bin/nvidia-container-toolkit -debug $@
 ```
 
+Make sure the hook shell is executable:
+
+```
```

Comment:
Add `sh` to specify the code block language.

Reply:
Thanks, I will update "```" to "```sh"